src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -278,7 +278,7 @@ namespace CondenserLoopTowers {
             state.dataCondenserLoopTowers->towers(TowerNum).Name = AlphArray(1);
             state.dataCondenserLoopTowers->towers(TowerNum).thisTowerNum = TowerNum;
             state.dataCondenserLoopTowers->towers(TowerNum).TowerType = cCurrentModuleObject;
-            state.dataCondenserLoopTowers->towers(TowerNum).TowerType_Num = DataPlant::TypeOf_CoolingTower_SingleSpd;
+            state.dataCondenserLoopTowers->towers(TowerNum).TowerType_Num = DataPlant::PlantEquipmentType::CoolingTower_SingleSpd;
```

Comment:
There is no need to store both `TowerType` and `TowerType_Num`. Store `TowerType` (the `enum`) and then lookup the name in the `constexpr std::array<std::string_view>` when you need it, which really should not be often. EnergyPlus should only deal with strings during setup and reporting.